core/src/main/java/hudson/model/operations/Operation.java

```diff
@@ -0,0 +1,48 @@
+package hudson.model.operations;
```

Comment:
If it is a new interface, it should be created in the jenkins.* package tree instead